sql/telemetry/fxa_content_auth_oauth_events_v1/view.sql

```diff
@@ -0,0 +1,89 @@
+CREATE OR REPLACE VIEW
```

Comment:
This file needs to be defined under `templates/` rather than `sql`; see https://github.com/mozilla/bigquery-etl#contributing ; this instructions may be new since you last contributed.

Reply:
oops will fix